src/providers/pipedrive.ts

```diff
@@ -0,0 +1,63 @@
+import { OAuthConfig, OAuthUserConfig } from "./oauth"
+
+export interface PipedriveProfile {
+  success: boolean;
+  data:    Data;
+}
+
+interface Data {
+  id:                     number;
+  name:                   string;
+  default_currency?:      string;
+  locale?:                string;
+  lang?:                  number;
+  email:                  string;
+  phone?:                 string;
+  activated?:             boolean;
+  last_login?:            Date;
+  created?:               Date;
+  modified?:              Date;
+  signup_flow_variation?: string;
+  has_created_company?:   boolean;
+  is_admin?:              number;
+  active_flag?:           boolean;
+  timezone_name?:         string;
+  timezone_offset?:       string;
+  role_id?:               number;
+  icon_url?:              string;
+  is_you?:                boolean;
+  company_id?:            number;
+  company_name?:          string;
+  company_domain?:        string;
+  company_country?:       string;
+  company_industry?:      string;
+  language?:              Language;
+}
+
+interface Language {
+  language_code?: string;
+  country_code?:  string;
+}
```

Comment:
Could you have this as a single interface? Also, if you have it configured, could you run Prettier on the file for consistent formatting with the rest of the codebase? :pray:

Reply:
I've fixed the interface. however, I don't have Prettier configured, could you run for me, please?

Reply:
@estevanjantsk I run `prettier` for you on the file 👍🏽 , for next time, if you already did `npm install` on the cloned project just run `npm run lint:fix` and it'll format the file for you 💅🏽

I think this provider is ready to go then @balazsorban44  🤞🏽 ?